src/style.css

```diff
@@ -1 +1,28 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  display: inline-block;
+  justify-content: center;
```

Comment:
this property works only in flex-container

Reply:
✅